.eslintrc.json

```diff
@@ -44,6 +44,9 @@
     "@typescript-eslint/ban-ts-comment": "off",
     "react/prop-types": "warn",
     "no-param-reassign": "warn",
+    "no-use-before-define": "warn",
+    "no-unused-expressions": "warn",
+    "camelcase": "warn",
```

Comment:
can i check why these were added

Reply:
we have a lot of cases which causes the lint to fail, i had to make them as warns for ci to pass

we should of course fix these issues in a separate PR, as there has been a lot of accumulated lint/format issues